### DISCUSSION TITLE
specaccum function for the accumulation data, and what function for the species shared in the sample?

### DISCUSSION THREAD

**Discussion by LuffyLuffy** [vegandevs/vegan#545] [Q&A]
Hi, there,
<img width="473" alt="image" src="https://user-images.githubusercontent.com/15357865/199774837-eaa2ff28-1a3e-42e2-8795-b14a823b5c48.png">
Species accumulation boxplot for each sample based on the genus level ASVs for each sample. A single red box reflects the total number of species in the sample, and the total red box forms a cumulative curve, reflecting the rate of new species emergence under continuous sampling; a single green box reflects the number of species shared in the sample.

I wonder if there is a function to  calculate shared species in sample.

Any hint on this would be highly appreciated.

**Comment by jarioksa** [comment]
What do you mean with "shared species"? Species that occur in *every* sampling unit, or species that occur in at least *two* sampling units, or species that occur both in the sampled and non-sampled units?

**Reply by LuffyLuffy** [reply]
> What do you mean with "shared species"? Species that occur in _every_ sampling unit, or species that occur in at least _two_ sampling units, or species that occur both in the sampled and non-sampled units?
Thanks for your reply.

I try to randomly sample 2-(totalsample_number-1) samples 10 times , and calculate the presence of species, then make a plot.
